pagerduty/event_orchestration_path.go

```diff
@@ -143,5 +152,5 @@ func (s *EventOrchestrationPathService) Update(id string, pathType string, orche
 		return nil, nil, err
 	}
 
-	return v.OrchestrationPath, resp, nil
+	return v, resp, nil
```

Comment:
This is the main change - I changed the signature of the `Update` method to return the entire endpoint response instead of just the `orchestration_path` property.